tests/components/google_generative_ai_conversation/test_config_flow.py

```diff
@@ -0,0 +1,118 @@
+"""Test the Google Generative AI Conversation config flow."""
+from unittest.mock import patch
+
+from google.api_core.exceptions import ClientError
+from google.rpc.error_details_pb2 import ErrorInfo
+import pytest
+
+from homeassistant import config_entries
+from homeassistant.components.google_generative_ai_conversation.const import (
+    CONF_CHAT_MODEL,
+    CONF_TOP_K,
+    CONF_TOP_P,
+    DEFAULT_CHAT_MODEL,
+    DEFAULT_TOP_K,
+    DEFAULT_TOP_P,
+    DOMAIN,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from tests.common import MockConfigEntry
+
+
+async def test_form(hass: HomeAssistant) -> None:
+    """Test we get the form."""
+    # Pretend we already set up a config entry.
+    hass.config.components.add("google_generative_ai_conversation")
+    MockConfigEntry(
+        domain=DOMAIN,
+        state=config_entries.ConfigEntryState.LOADED,
+    ).add_to_hass(hass)
```

Comment:
Why is the loaded state important?

Reply:
Same. This was copied from openai_conversation integration.